examples/README.md

```diff
@@ -15,6 +15,11 @@ This page contains a list of example codes written with Optuna.
 * [LighGBM](./lightgbm_simple.py)
 * [XGBoost](./xgboost_simple.py)
 
+### An example where a objective function uses additional arguments
```

Comment:
'a objective' is a typo for 'an objective'.

Reply:
Oops..., I fixed it.